models/behavior.js

```diff
@@ -0,0 +1,9 @@
+class Behavior {
```

Comment:
Ah ok something like how migrations and Models are named in rails convention. That sounds ok to me.